packages/errors/src/errors.ts

```diff
@@ -1052,6 +1052,12 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/migration-guide`
   },
+  EXPERIMENTAL_SESSION_SUPPORT_REMOVED: () => {
+    return errTemplate`\
+        The ${fmt.highlight(`experimentalSessionSupport`)} configuration option was removed in ${fmt.cypressVersion(`9.6.0`)} and replaced with ${fmt.highlight(`experimentalSessionAndOrigin`)}. Please update your config to use ${fmt.highlight(`experimentalSessionAndOrigin`)} instead.
+        
```

Comment:
```suggestion

```

Reply:
I think the space here is intentional looking at the original code and the html snapshots

Reply:
I think the space here is intentional looking at the 9.x branch as well as the snapshot templates